### PR TITLE
main: add SIE/SIA selection

### PR DESCRIPTION
**Pull request by huaracheguarache**
Add preliminary functionality to select Sea Ice Extent or Sea Ice Area.

Signed-off-by: Michael Yartys <michael.yartys@protonmail.com>

### CODE REVIEW

File: bokeh-app/main.py [hunk 1]
```diff
@@ -103,61 +103,68 @@ def get_data(url):
...
-urls_dict = {'South':'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/sh/osisaf_sh_sie_daily.nc', 
-        'North':'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/nh/osisaf_nh_sie_daily.nc'}
-
-df = get_data(urls_dict['North'])
-
-years = pn.widgets.MultiChoice(
-    name="Years:", options=list(df.columns), margin=(0, 20, 0, 0)
-)
-
+urls_dict = {"Sea Ice Extent":{'North':'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/nh/osisaf_nh_sie_daily.nc',
```
Comment (TomLav) [inline bokeh-app/main.py:143]: This is fine for now, but in the future (10-20 regions) we will need a better way to fill this urls_dict(), e.g. from a list of supported regions.

Comment (TomLav) [reply bokeh-app/main.py:143]: I am good with this PR, but it would be great if you could give it a look as well @epifanio :-)

Comment (epifanio) [reply bokeh-app/main.py:143]: looks good to me as well -- regarding the selection of different regions - if each region refers to a different NetCDF resource - then this "same" code will work just fine with a few modifications.

**Comment by epifanio** [comment]
I am merging thi. We do not have a CI for the app,, as it is hard to test a User Interface -- however, after merging the mybinder link will rebuild the image using the latest commit.. so  will be good if @huaracheguarache @TomLav we test the mybinder bokeh link to see if the app works as expected after the merge

**Pull request merged**